zombies/src/main/java/org/phantazm/zombies/kill/BasicPlayerKills.java

```diff
@@ -2,13 +2,24 @@
 
 import org.jetbrains.annotations.NotNull;
 import org.phantazm.mob.PhantazmMob;
+import org.phantazm.stats.zombies.ZombiesPlayerMapStats;
+
+import java.util.Objects;
 
 public class BasicPlayerKills implements PlayerKills {
+
+    private final ZombiesPlayerMapStats stats;
+
     private int kills = 0;
 
+    public BasicPlayerKills(@NotNull ZombiesPlayerMapStats stats) {
+        this.stats = Objects.requireNonNull(stats, "stats");
+    }
+
     @Override
     public void onKill(@NotNull PhantazmMob mob) {
         ++kills;
+        stats.setKills(stats.getKills() + 1);
```

Comment:
would be nice to also record the type of mob killed

Reply:
I could do it, but is it really useful?
I know Hypixel records the info but it seems fairly useless info to me and I don't think we should be recording every single thing

Reply:
what is the point of statistics if not recording useless info?
(in all seriousness i don't care too much either way)